### DISCUSSION TITLE
I found a new bypass what to see:)

### DISCUSSION THREAD

**Discussion by tran733** [3kh0/ext-remover#290] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
https://docs.google.com/document/d/1PyjkcA4CVbXYX4gYIh1gM6wPz1SDFr179ReIonT5Q5k/edit#

**Comment by tran733** [comment]
It unblocks the blocker extension of any blocker sites. It works

**Reply by bartholemuew** [reply]
but how does it do that

**Reply by tran733** [reply]
It logs you out and logs you back in but without the blocker extension ( disabled)

**Reply by bartholemuew** [reply]
but how does that work in the first place

**Reply by landodestroyer** [reply]
When I run the bookmarks all websites wont send data aside from chrome websites

**Comment by skoopy-star** [comment]
this works i approve 👍

**Comment by Huntlb** [comment]
They blocked tinyurl, what's the unshortened form of it?

**Reply by tran733** [reply]
https://accounts.google.com/signin/v2/identifier?hl=en&continue=https%3A%2F%2Fwww.google.com%2F&ec=GAlAmgQ&flowName=GlifWebSignIn&flowEntry=AddSession

**Reply by ghost** [reply]
W Bypass

**Comment by ItsLumine** [comment]
This actually works, but unfortunately FortiGuard filtered somethings

**Reply by ghost** [reply]
yea i still have go gaurdian and securly

**Reply by ghost** [reply]
idk how to remove them

**Reply by skoopy-star** [reply]
I don’t think you have to remove the extension because I just resigned in the extension was still there but didn’t block nothing

**Reply by tran733** [reply]
It doesn't remove the extensions. It just confuses the program so it disables the extension for that session. ( when you log out it deletes everything)

**Reply by skoopy-star** [reply]
does it also “confuse” dyknow

**Reply by 3kh0** [reply]
@skoopy-star yes it will, I also recommend using my about blank embedder

**Reply by skoopy-star** [reply]
what blank embedder and what does it do

**Reply by skoopy-star** [reply]
nvm I’m stupid how do I delete replies

**Reply by skoopy-star** [reply]
help

**Reply by bartholemuew** [reply]
> help

bro is confused 💀

**Comment by frogmasamune** [comment]
bruh you just plagiarized this exploit 
![Screenshot 2022-12-04 at 10 31 56 AM](https://user-images.githubusercontent.com/117880221/205508950-27f3757e-ac6f-48fc-9429-f9688fddaaf9.png)

**Reply by skoopy-star** [reply]
can you send discord invite

**Reply by tran733** [reply]
Plagiarized in the wrong word. I said I "found" the exploit not created.  Its in the topic discussion.

**Reply by frogmasamune** [reply]
found implies the same thing, and you never gave credit in this post or the doc

**Reply by 3kh0** [reply]
> can you send discord invite

https://discord.gg/unblock

**Reply by tran733** [reply]
![image](https://user-images.githubusercontent.com/67661966/205650923-960a4755-3685-4e4f-9e30-448508f602b8.png)
I never took the work as my own and I just said I found it online. I just wanted to share it to people not take it as my own work.

**Reply by fnfgo** [reply]
😭 it works unless you use CCPS chromebook which unfortunatly im on  😭 😭 😭 😭

**Reply by frogmasamune** [reply]
Sorry, my bad. But even though your intentions weren't bad, you could've just screenshotted the post from discord and shared it.

**Comment by tran733** [comment]
I just sent the original google docs. I didn't even type a single word.

**Comment by rwg100** [comment]
for dyknow the extension still shows up but it just doesnt work? will it still show student thumbnails or is it literally just decorative after this exploit. also are all extensions disabled?

**Reply by tran733** [reply]
Dyknow will be confused if this is a school account or a home account so it would be unable to work.

**Reply by rwg100** [reply]
will all of the extensions be disabled though?

**Reply by ItsLumine** [reply]
Every extension is disabled after you do the process

**Comment by bartholemuew** [comment]
so does this reset every time you restart?

**Reply by tran733** [reply]
Yes resets everytime you logout

**Comment by 6829ASCS** [comment]
Whats the alternate link for the SignOut bookmark? It's blocked for me..

**Reply by 3kh0** [reply]
https://accounts.google.com/signin/v2/identifier?hl=en&continue=https%3A%2F%2Fwww.google.com%2F&ec=GAlAmgQ&flowName=GlifWebSignIn&flowEntry=AddSession

**Comment by Aka-but-nice** [comment]
You fucked up

**Reply by Alex2Mayhem** [reply]
> You fucked up

Who and how?

**Reply by Aka-but-nice** [reply]
I'm talking to 6829ASCS
